test/integration/api_operations/list_test.rb

```diff
@@ -28,7 +29,8 @@ def assert_resource_type(resource, elem)
   end
 
   [[Clever::District, 1], [Clever::School, 3], [Clever::Teacher, 89],
-   [Clever::Student, 1004], [Clever::Section, 379], [Clever::Event, 7]].each do |test_data|
+   [Clever::Student, 1004], [Clever::Section, 379], [Clever::Event, 7],
+   [Clever::SchoolAdmin, 1]].each do |test_data|
```

Comment:
Yes, because the DistrictAdmin tests would fail since your district admin API does not support count or limit.